apps/ledger-live-mobile/src/components/DeviceActionProgress.tsx

```diff
@@ -2,7 +2,7 @@ import React, { memo } from "react";
 import { ProgressLoader, Text } from "@ledgerhq/native-ui";
 
 type Props = {
-  progress?: number;
+  progress?: number | null;
```

Comment:
why ?

Reply:
why not? without it we have type errors because progress can be null